### ISSUE TITLE
(make local dirs and public dirs the same)solution dir paths don't work

### ISSUE THREAD

**Issue by buus2** [neptune-ml/minerva#38]
**Fashion mnist**

These work:
- in `neptune.yaml`: `solution_dir: resources/fashion_mnist/solution`
- command
`python main.py -- dry_run --problem fashion_mnist --train_mode False`
or
`neptune run -- dry_run --problem fashion_mnist --train_mode False`

This doesn't work:
- in `neptune.yaml`: `solution_dir: /public/minerva/resources/fashion_mnist/solution`
- command
```
neptune send \
--environment keras-2.0-gpu-py3 \
--worker gcp-gpu-medium \
-- dry_run --problem fashion_mnist --train_mode False
```

Please make resources on github repo and neptune's public exactly the same.

**Whales**

This doesn't work:
- in `neptune.yaml`:
```
data_dir: resources/whales/data
solution_dir: resources/whales/solution
```
- command
`python main.py -- dry_run --problem whales --train_mode False`

Error: `ValueError: Specified solution_dir is missing 'transformers' directory. Use dry_run with train_mode=True or specify the path to trained pipeline`.

It seems that automatic subproblem inference doesn't reach there.

**Comment by jakubczakon** [comment]
Done, waiting for PR to merge.

**Comment by kamil-kaczmarek** [comment]
@jakubczakon @buus2 merged #49
